backend/schemas.py

```diff
@@ -252,7 +252,7 @@ def incident_to_orm(incident: CreateIncidentSchema) -> Incident:
     return Incident(**orm_attrs)
 
 
-def incident_orm_to_json(incident: Incident) -> dict:
+def incident_orm_to_json(incident: Incident) -> dict[str, Any]:
```

Comment:
There's something about this change that it didn't like...
```
________________________ ERROR collecting test session _________________________
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:170: in exec_module
    exec(co, module.__dict__)
backend/tests/conftest.py:3: in <module>
    from backend.api import create_app
backend/api.py:11: in <module>
    from backend.schemas import spec
backend/schemas.py:[25](https://github.com/codeforboston/police-data-trust/actions/runs/7915797626/job/21608382167?pr=325#step:7:26)5: in <module>
    def incident_orm_to_json(incident: Incident) -> dict[str, Any]:
E   TypeError: 'type' object is not subscriptable

=========================== short test summary info ============================
ERROR  - TypeError: 'type' object is not subscriptable
```

Reply:
Ah lol forgot cloud uses 3.8. Can we downgrade the dev container in the meantime?